cli/groups.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	mfxsdk "github.com/mainflux/mainflux/pkg/sdk/go"
+	"github.com/spf13/cobra"
+)
+
+var cmdGroups = []cobra.Command{
+	cobra.Command{
+		Use:   "create",
```

Comment:
What is the full command? `group create ...`? Would it not be confusing when we add other entities groups (things, channels)?